implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -119,6 +121,9 @@ pub enum OckamSubcommand {
     #[clap(display_order = 900, help_template = HELP_TEMPLATE, name = "token")]
     GenerateEnrollmentToken(GenerateEnrollmentTokenCommand),
 
+    /// Create forwarders
+    Forwarder(ForwarderCommand),
```

Comment:
```suggestion
    /// Create forwarders
    #[clap(display_order = 900, help_template = HELP_TEMPLATE)]
    Forwarder(ForwarderCommand),
```

You forgot the `clap` attribute invocation here. I _think_ it's important anyway :sweat_smile:

Reply:
Yeah, now that you mention it I think it renders the help template without the attribute. I'll open a separate PR removing them from subcommands because I'm pretty sure it's inherited from the top-level command.